src/features/user/Account/History.tsx

```diff
@@ -60,6 +78,46 @@ export default function History({
     }
   }, [paymentHistory]);
 
+  const handleIssueReceipts = async () => {
+    setLoading(true);
+    try {
+      const res = await postAuthenticatedRequest(
+        '/app/taxReceipts',
```

Comment:
Add expected response type `T` for `postAuthenticatedRequest`

Reply:
Added here https://github.com/Plant-for-the-Planet-org/planet-webapp/pull/1677/commits/28c77d939c7e6826d360a5c9fdc8cf9528e9dfd0